t/script/release/skip_broken_files.t

```diff
@@ -0,0 +1,35 @@
+use strict;
+use warnings;
+
+use Archive::Tar;
+use HTTP::Request;
+use LWP::UserAgent;
+use MetaCPAN::Script::Release;
+use Test::More;
+
+#use Test::RequiresInternet ( 'https://metacpan.org/' => 443 );
```

Comment:
`use Test::RequiresInternet ( 'metacpan.org' => 443 );`

See the SYNOPSIS https://metacpan.org/pod/Test::RequiresInternet#SYNOPSIS